components/governance/org.wso2.carbon.governance.taxonomy.ui/src/main/resources/web/taxonomy/save_taxonomy-ajaxprocessor.jsp

```diff
@@ -40,9 +38,7 @@
             }
         }
 
-    } catch (RemoteException e) {
-        error = e.getMessage().replaceAll(">", "&gt;").replaceAll("<", "&lt;");
-    } catch (TaxonomyServicesRegistryExceptionException e) {
+    } catch (Exception e) {
```

Comment:
Since we can't catch specific type exceptions inside jsp, used exception super class.  If we use specific exception type , there will be class not found exception for that specific exception